mllib/src/main/scala/org/apache/spark/mllib/evaluation/RegressionMetrics.scala

```diff
@@ -27,11 +27,17 @@ import org.apache.spark.sql.DataFrame
 /**
  * Evaluator for regression.
  *
- * @param predictionAndObservations an RDD of (prediction, observation) pairs.
+ * @param predictionAndObservations an RDD of (prediction, observation) pairs,
+ * @param throughOrigin True if the regression is through the origin. For example, in linear
+ *                      regression, it will be true without fitting intercept.
  */
 @Since("1.2.0")
 class RegressionMetrics @Since("1.2.0") (
-    predictionAndObservations: RDD[(Double, Double)]) extends Logging {
+    predictionAndObservations: RDD[(Double, Double)], throughOrigin: Boolean)
```

Comment:
@iyounus ah we have two tiny problems to sort then this can be merged. First the `@Since` annotation is wrong here; it needs to move to the new constructor, and this needs a new "since 2.0.0" annotation on the main constructor. Also, in the scaladoc, `predictionAndObservations` ends with a comma now but should be left unchanged